controllers/test_util.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2021 Red Hat, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !release
+
+package controllers
+
+import "net/http"
+
```

Comment:
Why do we need these methods?

Reply:
Used in tests, for example here: https://github.com/redhat-appstudio/service-provider-integration-oauth/pull/9/files#diff-1aa5fd38c7bc23ab9eb38993931360bb3a906c5342cc802c8f9ffc6b362a20f4R84

Reply:
Used to reinterpret a function as an `http.RoundTripper`